packages/perseus-core/src/index.ts

```diff
@@ -27,3 +27,6 @@ export {PerseusError} from "./error/perseus-error";
 export * from "./data-schema";
 
 export {pluck, mapObject} from "./utils/objective_";
+
+export * as expressionUpgrade from "./widgets/expression";
```

Comment:
* I think this could simply be called `expression` (it's not strictly related to upgrades... well, maybe kinda... meh... it's a name)
* I would feel more comfortable if the pieces we're exporting per-widget had some type that defined what we export. Kinda like a subset of `WidgetExports<T>`. 🤔

Reply:
I think that makes sense if long-term we want to move the scorers into core.